src/account/Settings.tsx

```diff
@@ -55,222 +58,167 @@ import {
-    walletConnectEnabled: v1,
-    supportedBiometryType: supportedBiometryTypeSelector(state),
-    shouldShowRecoveryPhraseInSettings: shouldShowRecoveryPhraseInSettingsSelector(state),
-    hapticFeedbackEnabled: hapticFeedbackEnabledSelector(state),
-    decentralizedVerificationEnabled: decentralizedVerificationEnabledSelector(state),
-  }
-}
-
-const mapDispatchToProps = {
-  revokeVerification,
-  setNumberVerified,
-  resetAppOpenedState,
-  setAnalyticsEnabled,
-  toggleBackupState,
-  devModeTriggerClicked,
-  setRequirePinOnAppOpen,
-  setPincodeSuccess,
-  setSessionId,
-  clearStoredAccount,
-  hapticFeedbackSet,
-}
-
-interface State {
-  showAccountKeyModal: boolean
-  showRevokeModal: boolean
-}
-
-export class Account extends React.Component<Props, State> {
-  componentDidMount = () => {
-    const sessionId = ValoraAnalytics.getSessionId()
-    if (sessionId !== this.props.sessionId) {
-      this.props.setSessionId(sessionId)
+import { useRevokeCurrentPhoneNumber } from 'src/verify/hooks'
+import { selectSessions as selectSessionsV1 } from 'src/walletConnect/v1/selectors'
+import { selectSessions as selectSessionsV2 } from 'src/walletConnect/v2/selectors'
+import { walletAddressSelector } from 'src/web3/selectors'
+
+type Props = NativeStackScreenProps<StackParamList, Screens.Settings>
+
+export const Account = ({ route }: Props) => {
+  const dispatch = useDispatch()
+  const { t } = useTranslation()
+
+  const revokeNumberAsync = useRevokeCurrentPhoneNumber()
+
+  const [showAccountKeyModal, setShowAccountKeyModal] = useState(false)
+  const [showRevokeModal, setShowRevokeModal] = useState(false)
+  const [showAccountResetModal, setShowAccountResetModal] = useState(false)
+
+  const sessionId = useSelector(sessionIdSelector)
+  const account = useSelector(walletAddressSelector)
+  const devModeActive = useSelector(devModeSelector)
+  const e164PhoneNumber = useSelector(e164NumberSelector)
+  const analyticsEnabled = useSelector(analyticsEnabledSelector)
+  const numberVerified = useSelector(phoneNumberVerifiedSelector)
+  const pincodeType = useSelector(pincodeTypeSelector)
+  const requirePinOnAppOpen = useSelector(getRequirePinOnAppOpen)
+  const preferredCurrencyCode = useSelector(getLocalCurrencyCode)
+
+  const { sessions: walletConnectV1Sessions } = useSelector(selectSessionsV1)
+  const { sessions: walletConnectV2Sessions } = useSelector(selectSessionsV2)
+
+  const { v1, v2 } = useSelector(walletConnectEnabledSelector)
+  const supportedBiometryType = useSelector(supportedBiometryTypeSelector)
+  const shouldShowRecoveryPhraseInSettings = useSelector(shouldShowRecoveryPhraseInSettingsSelector)
+  const hapticFeedbackEnabled = useSelector(hapticFeedbackEnabledSelector)
+  const decentralizedVerificationEnabled = useSelector(decentralizedVerificationEnabledSelector)
+  const currentLanguage = useSelector(currentLanguageSelector)
+
+  const walletConnectEnabled = v1 || v2
```

Comment:
this change resulted in one of the snapshots updating. previously, this was only looking at whether v1 was enabled. i can't remember why it was like this, but this flag shows / hides the connected applications. idk why we'd want to hide them if v1 is off but v2 is on. so i added it